azureproject/production.py

```diff
@@ -1,21 +1,19 @@
 import os
+import secrets
 
 # SECURITY WARNING: keep the secret key used in production secret!
-# Use this py command to create secret 
-# python -c 'import secrets; print(secrets.token_hex())'
-SECRET_KEY = os.getenv('AZURE_SECRET_KEY')
+SECRET_KEY = os.environ['SECRET_KEY'] if 'SECRET_KEY' in os.environ else secrets.token_hex()
 
-# Configure allowed host names that can be served and trusted origins for Azure Container Apps.
-ALLOWED_HOSTS = ['.azurecontainerapps.io'] if 'RUNNING_IN_PRODUCTION' in os.environ else []
-CSRF_TRUSTED_ORIGINS = ['https://*.azurecontainerapps.io'] if 'RUNNING_IN_PRODUCTION' in os.environ else []
 DEBUG = False
+ALLOWED_HOSTS = [os.environ['WEBSITE_HOSTNAME']] if 'WEBSITE_HOSTNAME' in os.environ else []
```

Comment:
This is going to cause a runtime error if the env variable doesnt exist. You need to either use os.environ.get() or os.getenv()